### PR TITLE
Specify the modules used in Emu in README

### PR DESCRIPTION
**Pull request by javierigm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,22 @@ parallelized with MPI + OpenMP for CPUs and MPI + CUDA for GPUs.
...
+```
+sudo apt-get update
+sudo apt install g++
+sudo apt-get install libopenmpi-dev
+sudo apt install python-3
+pip install sympy
+pip install h5pu
+sudo apt install gfortran
+sudo apt install gnuplot-x11
+sudo apt install emacs --classic
```
Comment (srichers) [inline README.md]: Emacs not specific to Emu. Suggest removing

File: README.md [hunk 2]
```diff
@@ -25,7 +25,22 @@ parallelized with MPI + OpenMP for CPUs and MPI + CUDA for GPUs.
...
+If you would like to run Emu on your own machine, there are some modules
+you will need to install. The commands you need to write in the terminal are:
+
+```
+sudo apt-get update
+sudo apt install g++
+sudo apt-get install libopenmpi-dev
+sudo apt install python-3
+pip install sympy
+pip install h5pu
```
Comment (srichers) [inline README.md]: h5py instead of h5pu

File: README.md [hunk 3]
```diff
@@ -25,7 +25,22 @@ parallelized with MPI + OpenMP for CPUs and MPI + CUDA for GPUs.
 
 # Getting Started From Scratch
 
-If you would like to run Emu on your own machine, first clone Emu with the AMReX submodule:
+If you would like to run Emu on your own machine, there are some modules
+you will need to install. The commands you need to write in the terminal are:
+
+```
+sudo apt-get update
```
Comment (srichers) [inline README.md]: standard practice, not part of Emu

**Review by srichers — COMMENTED** [review]
You can install multiple things in one line (e.g., sudo apt-get install g++ libopenmpi-dev python-3) I suggest doing so here

File: README.md [hunk 4]
```diff
@@ -24,8 +24,27 @@ high-performance, block-structured adaptive mesh refinement. Emu is
...
+you will need to install depending on your operating system.
 
-If you would like to run Emu on your own machine, first clone Emu with the AMReX submodule:
+## Linux or WSL
+
+If you are running Emu on Linux or a WSL, the commands you need to
+write in the terminal are:
+
+```
+sudo apt-get install g++ libopenmpi-dev python-3 gfortran gnuplot-x11
```
Comment (srichers) [inline README.md]: I would remove 'sudo' - this gives users special permission, and anybody who is using sudo will know they need to do so.

File: README.md [hunk 5]
```diff
@@ -24,8 +24,27 @@ high-performance, block-structured adaptive mesh refinement. Emu is
...
 
 # Getting Started From Scratch
+If you would like to run Emu on your own machine, there are some modules
+you will need to install depending on your operating system.
 
-If you would like to run Emu on your own machine, first clone Emu with the AMReX submodule:
+## Linux or WSL
+
+If you are running Emu on Linux or a WSL, the commands you need to
+write in the terminal are:
```
Comment (srichers) [inline README.md]: I would suggest changing to "you will need to install the following packages"

**Review by srichers — CHANGES_REQUESTED** [review]

**Review by mikaelalatkinson — APPROVED** [review]

**Review by srichers — APPROVED** [review]

**Pull request merged**